src/Umbraco.Web.UI.Client/vwd.webinfo

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0"?>
```

Comment:
No need to commit this file.

Reply:
Well, actually, I have no idea what that file is :-D

What am I supposed to do with it? Just delete it?

Reply:
Haha, then don't commit it! 😉 

I think it's Visual Studio "trying" to be helpful. :) Yep, just delete it!

Reply:
Sorry, Umbraco is the first project I'm contributing to via Github. It's not that easy to understand quickly :-D But I'd like to contribute more to Umbraco, so I'll try to pay attention to these kind of things.

Reply:
Not to worry, we're here to help and so far you've done great! 👍